pkg/errs/errors.go

```diff
@@ -0,0 +1,8 @@
+package errs
+
+import "errors"
+
+var DbError = errors.New("unexpected database error")
+var ContentType404 = errors.New("content type not found")
+var Column404 = errors.New("column not found")
+var Content404 = errors.New("content not found")
```

Comment:
have this as 
`var (`
 `// the errors go here`
`)`

Reply:
Done, check the last commit